opensearch/src/main/java/org/opensearch/sql/opensearch/storage/scan/OpenSearchPagedIndexScanBuilder.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.sql.opensearch.storage.scan;
+
+import lombok.EqualsAndHashCode;
+import org.opensearch.sql.storage.TableScanOperator;
+import org.opensearch.sql.storage.read.TableScanBuilder;
+
+/**
+ * Builder for a paged OpenSearch request.
+ * Override pushDown* methods from TableScanBuilder as more features
+ * support pagination.
+ */
+public class OpenSearchPagedIndexScanBuilder extends TableScanBuilder {
```

Comment:
This class just a holder, and seems do nothing?

Reply:
Nice finding. Yes, it is. It was simplified so it became just a holder.